src/share-command.js

```diff
@@ -90,6 +90,10 @@ ShareCommand.prototype._defaultFn = function () {
         }, 100);
 
         function hideShare(ev) {
+            // prevent space from scrolling the page
+            if (ev.type === "keyup" && ev.which === 32) {
+                ev.preventDefault();
```

Comment:
This and similar in share-menu.js are starting to make the case for an aria util, probably in streamhub-ui.